diesel/src/pg/types/mod.rs

```diff
@@ -11,4 +12,6 @@ pub mod sql_types {
     pub type SmallSerial = ::types::SmallInt;
     pub type Serial = ::types::Integer;
     pub type BigSerial = ::types::BigInt;
+    pub type Uuid = super::uuid::uuid::Uuid;
+    pub type Bytea = ::types::Binary;
```

Comment:
I don't see any reason for us to add this alias. I prefer to use the generic name whenever possible (`Serial` actually does imply something different than `Integer` and will change to accommodate this in the future)